src/pages/Swap/index.tsx

```diff
@@ -410,125 +130,25 @@ export default function Swap() {
-                          onSwitchTokens()
-                        }}
-                      >
-                        <ArrowWrapper
-                          clickable={!loading}
-                          data-testid="switch-tokens-button"
-                          className={loading ? 'rotate' : ''}
-                        >
-                          <SwapIcon />
-                        </ArrowWrapper>
-                      </SwitchTokensAmountsContainer>
-                    </SwitchIconContainer>
-                    <CurrencyInputPanel
-                      value={formattedAmounts[Field.OUTPUT]}
-                      onUserInput={handleTypeOutput}
-                      onMax={handleMaxInput(Field.OUTPUT)}
-                      currency={currencies[Field.OUTPUT]}
-                      onCurrencySelect={handleOutputSelect}
-                      otherCurrency={currencies[Field.INPUT]}
-                      fiatValue={fiatValueOutput}
-                      priceImpact={priceImpact}
-                      isFallbackFiatValue={isFallbackFiatValueOutput}
-                      maxAmount={maxAmountOutput}
-                      showCommonBases
-                      disabled={isInputPanelDisabled}
-                      id="swap-currency-output"
-                    />
-                  </AutoColumn>
 
-                  <TradeDetails
-                    show={!showWrap}
-                    loading={loading}
-                    trade={trade}
-                    bestPricedTrade={bestPricedTrade}
-                    showAdvancedSwapDetails={showAdvancedSwapDetails}
-                    setShowAdvancedSwapDetails={setShowAdvancedSwapDetails}
-                    recipient={recipient}
-                  />
-                  <SwapButtons
-                    wrapInputError={wrapInputError}
-                    showApproveFlow={showApproveFlow}
-                    userHasSpecifiedInputOutput={userHasSpecifiedInputOutput}
-                    approval={approval}
-                    setSwapState={setSwapState}
-                    priceImpactSeverity={priceImpactSeverity}
-                    swapCallbackError={swapCallbackError}
-                    wrapType={wrapType}
-                    approvalSubmitted={approvalsSubmitted[currentTradeIndex]}
-                    currencies={currencies}
-                    trade={trade}
-                    swapInputError={swapInputError}
-                    swapErrorMessage={swapErrorMessage}
-                    loading={loading}
-                    onWrap={onWrap}
-                    approveCallback={approveCallback}
-                    handleSwap={handleSwap}
-                    handleInputSelect={handleInputSelect}
-                    wrapState={wrapState}
-                    setWrapState={setWrapState}
-                  />
-                </AutoColumn>
-              </Wrapper>
-            </AppBody>
+              <ButtonGroup mb={[3, 0]}>
```

Comment:
If we are cleaning swap page I would also move ButtonGroup to a separate file

Reply:
Yes, that's a nice suggestion.

Reply:
Done.